packages/teleterm/src/ui/services/docs/docs.ts

```diff
@@ -184,23 +189,41 @@ export default class DocumentService extends Store<State> {
     return useStore(this).state;
   }
 
-  addNewTerminalShellDocument() {
-    const doc: Document = {
-      uri: uris.getUriPty({ sid: unique() }),
-      title: 'Terminal',
-      kind: 'terminal_shell',
-    };
-
-    this.add(doc);
-
-    return doc;
-  }
-
   changeIndex(oldIndex: number, newIndex: number) {
     const doc = this.state.docs[oldIndex];
     const newDocs = [...this.state.docs];
     newDocs.splice(oldIndex, 1);
     newDocs.splice(newIndex, 0, doc);
     this.setState({ ...this.state, docs: newDocs });
   }
+
+  openNewDocument() {
```

Comment:
`openNewDocument` -> `openNewTerminal` (as this method name is less generic, dedicated specifically for terminals)